README.md

```diff
@@ -71,22 +67,31 @@ $ cargo install grpcio-compiler
 $ protoc --rust_out=. --grpc_out=. --plugin=protoc-gen-grpc=`which grpc_rust_plugin` example.proto
 ```
 
+
+### Option 2 - Programmatic Generation
+
+Programmatic generation can be used to generate Rust modules from proto files
+via your `build.rs` by using [protoc-grpcio](https://crates.io/crates/protoc-grpcio).
```

Comment:
You don't need to break the line because you are just writing a paragraph.

Reply:
The line is break because it's too long otherwise. Actually it's a feature of Markdown to break a long line into shorter ones and generate the same output.

Reply:
Got it. Thanks.